src/components/Home/Home.tsx

```diff
@@ -27,18 +41,67 @@ export function Home() {
       <br />
     </>
   ) : null;
+
+  const handleSubmit = () => {
+    const postalCodeRegex = /[a-zA-Z][0-9][a-zA-Z](-| |)[0-9][a-zA-Z][0-9]/;
+    if (postalCodeRegex.test(postalCode)) {
+      setRenderCards(true);
+      setShowPostalPrompt(false);
+    } else {
+      setShowInvalidPostal(true);
+    }
+  };
+  const afterPostalEntryMarkup = renderCards ? (
+    <>
+      <TextStyle variation="subdued">
+        Here&apos;s what&apos;s happening with vaccinations in Ottawa
+      </TextStyle>
+      <br />
+      <br />
+      {dismissableBannerMarkup}
+      <PharmacyList />
+    </>
+  ) : null;
+
+  const postalLabelMarkup = showInvalidPostal
```

Comment:
On error, we appear to not be calling out the issue stylistically or via a semantic accessible markup. We should set the input's `aria-invalid` attribute  to `true`. Also add `aria-describedby` to the input on error, and reference the error element:

```
<TextField name="postal"
		id="postal"
                aria-invalid
		aria-describedby="postalError">
		<span id="postalError">You have entered an invalid postal code. Please enter a valid one in this format: K2T 0E5</span>

Reply:
Is the way I have it set up with postalLabelMarkup not valid? If a postal is invalid showInvalidPostal is true and shows the message in the title of the textbox that they entered an invalid postal code.

Reply:
The label is no longer describing its related control, which might be confusing. I believe you can add the error message INSIDE the label as additional information, but the label should still be responsible for indicating what data the control is responsible for.

Also, the error isn't styled to indicate that the control is in an error state. Error stylings would include things like red text, a red outline, a big ol' X... Those visual cues are helpful to everyone, but especially those with visual and cognitive impairments.